apps/ledger-live-desktop/src/renderer/families/cosmos/UndelegationFlowModal/types.ts

```diff
@@ -16,15 +16,15 @@ export type StepProps = {
   onRetry: () => void;
   onClose: () => void;
   openModal: OpenModal;
-  optimisticOperation: CosmosOperation | undefined;
+  optimisticOperation: Operation | undefined;
```

Comment:
No way to keep it as CosmosOperation ?

Reply:
See my previous answer [here](https://github.com/LedgerHQ/ledger-live/pull/4235#discussion_r1300376565) :slightly_smiling_face: